core/src/main/java/org/elasticsearch/search/rescore/QueryRescorerBuilder.java

```diff
@@ -78,6 +75,25 @@ public QueryRescorerBuilder(QueryBuilder<?> builder) {
     }
 
     /**
+     * Read from a stream.
+     */
+    public QueryRescorerBuilder(StreamInput in) throws IOException {
```

Comment:
I moved this up here so it could be next to the other constructor. I moved doWriteTo up here so it could be with the reading code.